src/styles/blocks/card.css

```diff
@@ -0,0 +1,87 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: var(--color-main-text);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  display: flex;
+  justify-content: center;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__code {
+  margin-bottom: 16px;
+  font-size: 10px;
+  color: var(--color-text-gray);
+}
+
+.card__review {
+  margin-bottom: 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__review-text {
+  font-size: 10px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__price-name {
+  font-size: 12px;
+  color: var(--color-text-gray);
+}
+
+.card__price-value {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.card__button {
+  width: 166px;
+  height: 40px;
```

Comment:
it's better not hardcode height for elements with content inside. Just use paddings